pkg/component/operator/audio/v0/task_extract_audio_segments_test.go

```diff
@@ -0,0 +1,60 @@
+package audio
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"testing"
+
+	"github.com/frankban/quicktest"
+	"github.com/go-audio/wav"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/instill-ai/pipeline-backend/pkg/component/base"
+)
+
+func TestExtractAudioSegments(t *testing.T) {
+	c := quicktest.New(t)
+	testCase := struct {
+		audioFile     string
+		segmentsFile  string
+		expectedCount int
+	}{
+		audioFile:     "testdata/voice.wav",
+		segmentsFile:  "testdata/voice-activity-segments.json",
+		expectedCount: 15,
+	}
+
+	// Load audio file
```

Comment:
I think good comments are for
1. Exported document
2. Reason why the code is like this
3. If it is hard to understand, add what the code does

The code explains itself well, so I'd remove the comments here.